plugin/main/src/kotlinx/benchmark/gradle/BenchmarksPlugin.kt

```diff
@@ -7,7 +7,7 @@ import org.gradle.util.*
 class BenchmarksPlugin : Plugin<Project> {
     companion object {
         const val PLUGIN_ID = "org.jetbrains.kotlinx.benchmark"
-        const val PLUGIN_VERSION = "0.4.8"
+        const val PLUGIN_VERSION = "0.4.255-SNAPSHOT"
```

Comment:
Is this value correctly overridden when we're building a build for deployment?

Reply:
Now there is a task that overrides the value during a release build.